docs/serving/using-auto-tls.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Enabling automatic TLS certificate provisioning"
+linkTitle: "Enabling auto TLS certs"
+weight: 64
+type: "docs"
+---
+
+If you install and configure cert-manager, you can configure Knative to
+automatically obtain new TLS certificates and renew existing ones.
+To learn more about using secure connections in Knative, see
+[Configuring HTTPS with TLS certificates](./using-tls-cert.md).
+
+## Before you begin
+
+You must meet the following prerequisites to enable automatic certificate
+provisioning:
+
+- The following must be installed on your Knative cluter:
+  - [Knative Serving version 0.6.0 or higher](../install/).
+  - [Istio with SDS, version 1.1 or higher](../install/installing-istio.md#installing-istio-with-SDS-to-secure-the-ingress-gateway).
+    Note: Currently, [Gloo](https://github.com/solo-io/gloo) is unsupported.
+  - [cert-manager version `0.6.1` or higher](./installing-cert-manager.md).
+- Your Knative cluster must be configured to use a
+  [custom domain](./using-a-custom-domain.md).
+
+
+## Enabling automatic certificate provisioning
+
+To enable Knative to automatically provision TLS certificates:
+
+1. Run the following commands to install the `networking-certmanager`:
+
+   ```shell
+   # KNATIVE_VERSION needs to be 0.6.0 or above.
+   KNATIVE_VERSION=0.6.0
+
+   kubectl apply --filename https://github.com/knative/serving/releases/download/v${KNATIVE_VERSION}/serving.yaml \
+   --selector networking.knative.dev/certificate-provider=cert-manager
+    ```
+
```

Comment:
Before the step of configuring the `config-certmanager` ConfigMap, I think we need to add an extra step about configuring `Cert-Manager`.
In this step, users need to follow the [cert-manager instructions](https://docs.cert-manager.io/en/latest/tasks/acme/configuring-dns01/index.html#supported-dns01-providers) to configure DNS01 providers according to their platform of DNS servers.
Specifically, for Google Cloud DNS, we need users to 1) [create DNS service account and upload the credential of DNS servers](https://github.com/knative/docs/blob/master/docs/serving/using-cert-manager-on-gcp.md#creating-a-cloud-dns-service-account) 2) [Create `ClusterIssuer`](https://github.com/knative/docs/blob/master/docs/serving/using-cert-manager-on-gcp.md#specifying-a-certificate-issuer)

Reply:
The configure DNS01 step is now a part of installing cert-manager (i moved it there as the final step since it was a necessary steps for all use cases) and that step of installing cert-manager is listed as a prerequisite for this task (therefore it should be done before the user gets to this point).

Reply:
Yes, you are right.